### ISSUE TITLE
Typography - Resizing Text

### ISSUE THREAD

**Issue by jinky32** [ThreeUK/digital-library.three.co.uk#8]
**Requirement**: Users should be able to resize text up to 200% and it remain readable without assistive technology.
It should be checked that up to this level of zoom responsive layouts do not cause problems. For example: truncated words; words overlapping with other content; or only one word of a sentence fitting on each line, causing the sentence to be displayed as a vertical column of text that is difficult to read.

**Why is this required?** Many users benefit from being able to enlarge the display, for example through the browser zoom settings, or browser ‘largest font’ settings.

**Comment by papafed** [comment]
We could basically ignore this requirement and have users control this with their browser's zoom control.
However it would be really nice if we actually provided users with a little text-size control in the header or footer that increased or decreased font sizes - this would allow us to control the reflow with CSS.

**Comment by jinky32** [comment]
@papafed agreed - the requirement is that as users use their browser zoom the text / line spacing etc means that the text is legible.   

Text control in header could be a nice feature too.
